docs/community.rst

```diff
@@ -139,17 +139,26 @@ Get started!
 
    Now you can make your changes locally.
 
-6. When you're done making changes, check that your changes pass the tests::
+6. When you're done making changes, check that your changes pass linting (requires python >= 3.6)::
+
+    tox -e lint
+
+7. Don't forget to check that your changes pass the tests::
 
     make tests
 
-7. Commit your changes and push your branch to GitHub::
+8. (Optional) Install a git hook::
+
+    therapist install
+
```

Comment:
looks like an extra line

Reply:
Don't you have to tell therapist to create the hook in your .git dir?